oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/query/async/ElasticResultRowAsyncIterator.java

```diff
@@ -25,20 +25,18 @@
 import org.apache.jackrabbit.oak.plugins.index.search.util.LMSEstimator;
```

Comment:
If i am not missing something, then these changes would convert every request to low level request, right? The issue says we should use it for critical operations.

Reply:
This change converts all standard queries to make low-level requests. Noncritical operations that are left out are: suggest and spellcheck. These still use the high-level client.